### ISSUE TITLE
Timeline: UI to select and display active timeline

### ISSUE THREAD

**Issue by philipgrimmett** [TerriaJS/terriajs#2624]
- Whenever a timeline is displayed on screen it should be clearly labeled with the title of its associated workbench item
- Whenever there are multiple time series data sets loaded, users need a control to select the 'active' timeline

Concept sketch:
![datepicker-concept-1 copy](https://user-images.githubusercontent.com/5292858/29101468-bd5ab39c-7cf5-11e7-9c01-a1887905cc55.jpg)

**Comment by RacingTadpole** [comment]
Yes, especially that second point - the first point is actually done already (see screenshot below), but could maybe be made more obvious.
![image](https://user-images.githubusercontent.com/1757858/29101601-b58818f2-7cf6-11e7-932f-1498f074de8e.png)

**Comment by AnaBelgun** [comment]
Some user feedback from DoE: 
- The play functionality does not only show the actual dates, and so is not very effective for data like Landsat. Is there anyway of only including dates with data in a particular location?
- The timeline does not have an indicator for when data is available – would be good to have a blue line on the timeline too

**Comment by philipgrimmett** [comment]
Design concept for toggling between timelines: https://invis.io/7BEIQJGD3#/264552860_with_Clouds

Other features:
Workbench date field, with the option to jump to the previous/next data point
Date picker (via workbench item or timeline)
